pages/blog/[post].js

```diff
@@ -35,6 +36,9 @@ const SlimContentSection = styled(ContentSection)`
       color: ${({ theme }) => theme.colors.white};
     }
   }
+    ${below.med`
```

Comment:
@tsidel these changes look great! So glad you gave it some mobile love! It looks like the linter is failing the build, my guess is from the indentation here. One thing we for sure need to get better documented... the build checks formatting of the changes through prettier and eslist though their libraries. It would be great to get your help to figure out those differences so we can get environment setup document to cover the validation we use. In the meantime, if you push this indentation back a level I think all will be good and we can get these changes in. 😄

Reply:
@saragibby Thanks for your notes. I think I've fixed it. I ended up having new errors popup that were more descriptive than before, so there's an additional file updated again, but just for spacing.  I'm still finding my way around git, hopefully I handled this commit correctly.